src/util/CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ libcvc5_add_sources(
   didyoumean.h
   divisible.cpp
   divisible.h
+  ff_val.cpp
+  ff_val.h
```

Comment:
The names of these files and the class don't conform to the naming scheme we have been using for theory values. I suggest to rename this in a more consistent manner, maybe `finite_field_value.(h|cpp)` and `class FiniteFieldValue`. 

Further, `ff_val.h` lists TODOs in the header that are not associated with an issue.

Reply:
Fine by me.